media/wav.ksy

```diff
@@ -0,0 +1,272 @@
+meta:
+  id: wav
+  file-extension: wav
+  title: Microsoft WAVE audio file
+  endian: le
+  license: BSD-3-Clause-Attribution
```

Comment:
You can change the license to any reasonable license, so long as it's not "you can sue John Byrd for screwing this up".